ccdaservice/winservice.js

```diff
@@ -0,0 +1,37 @@
+var isWin = /^win/.test(process.platform);
+var f = __dirname +''//serveccda.njs';
+
+if( isWin ){
+	var Service = require('node-windows').Service;
+	//f =  '\\xampp\\htdocs\\openemr\\services\\ccdaservice\\serveccda.njs';
```

Comment:
Also, out of curiosity. For this script and the one above, if I go this in a web browser, will they run?